masonry/src/contexts.rs

```diff
@@ -858,7 +858,9 @@ impl LayoutCtx<'_> {
     /// [`WidgetPod::paint_insets`]: crate::widget::WidgetPod::paint_insets
     pub fn set_paint_insets(&mut self, insets: impl Into<Insets>) {
         let insets = insets.into();
-        trace!("set_paint_insets {:?}", insets);
+        if self.global_state.trace.layout {
+            trace!("set_paint_insets {:?}", insets);
+        }
```

Comment:
I think it's okay to outright remove those logs rather than gating them. They're redundant with the log in the layout pass the gives the final layout.